packages/runtime/container-runtime/src/channelCollection.ts

```diff
@@ -523,15 +510,7 @@ export class ChannelCollection implements IFluidDataStoreChannel, IDisposable {
 			return false;
 		}
 
-		const handle = new FluidObjectHandle(
-			context,
-			internalId,
-			this.parentContext.IFluidHandleContext,
-		);
-		this.parentContext.addedGCOutboundRoute(
-			this.containerRuntimeHandle.absolutePath,
-			handle.absolutePath,
-		);
+		this.parentContext.addedGCOutboundRoute("/", `/${internalId}`);
```

Comment:
These are absolute paths.  But channelCollection could be nested, right?  So seems like we need to do something else here.  Also at the end of getGcData.

Probably just open another bug/task about gaps in the Nested DataStore story.

Reply:
Yeah this will need some work with nested channel collections. Same with other places where GC is notified. I believe we will need some generic solution like every layer that `addedGCOutboundRoute` goes through updates the `fromPath` similar to how `getGCData` works.